python2verilog/ir/context.py

```diff
@@ -76,7 +76,7 @@ class Context(GenericReprAndStr):
     instances: dict[str, Instance] = field(default_factory=dict)  # generator instances
 
     @classmethod
-    def empty(cls):
+    def from_validated(cls):
         """
         Creates an empty but valid context for testing purposes
         """
```

Comment:
Is the function name accurate to what the code does? I think `def empty_valid_context` is more appropriate

Reply:
changed to `empty_valid`